### ISSUE TITLE
Use BrainVISA interface to select Capsul pipeline

### ISSUE THREAD

**Issue by sapetnioc** [populse/capsul#302]

**Comment by denisri** [comment]
This is partly done but we can discuss what we want exactly:
- We can declare in Axon a process which is a wrapping of a Capsul process or pipeline - but we don't translate the internal pipeline structure to axon: we use it as a black box, or we can edit the pipeline structure in capsul.
- we can instantiate an axon process which is an automatic wrapping of a Capsul process or pipeline, even if it is not declared as an axon process:
  ```
  p = getProcess('capsul://deepsulci.sulci_labeling.capsul.labeling')
  ```

But we don't see in Axon GUI all capsul processes and pipelines if they are not redeclared in axon.
Do we want more ?

**Comment by denisri** [comment]
Another sub-topic to be handled: completion schemas.
The way to load Capsul3's completion schemas (and `ProcessSchema` subclasses) given a process/pipeline and the needed schemas (Bids, brainrvisa etc) is not clear yet. So far we load Morphologist schemas (hard-coded) because we only have those and they are currently part of Capsul3 (#328), but there is currently no mechanism to load additional schema parts for additional processes or toolboxes.

**Comment by sapetnioc** [comment]
Yes. There is a low level and not very user friendly mechanism that allows
to load a schema: record the corresponding Python module in the list of
modules to load in the config. But we have to work on the definition of
higher level objects such as toolboxes.

Le jeu. 23 nov. 2023 à 17:39, Denis Rivière ***@***.***> a
écrit :
